README.md

```diff
@@ -62,17 +62,23 @@ Confluence page (*DESC member only*).
 
 -  **DC2 "Object Catalogs"** \
    *by LSST DESC, compiled by Michael Wood-Vasey*
-   - `dc2_object_run1.1p`: coadd catalog for Run 1.1p
-   - `dc2_object_run1.1p_tract4850`: same as `dc2_coadd_run1.1p` but has only one tract (4850) for testing purpose / faster access
+   - `dc2_object_run1.2i`: static object catalog for Run 1.2i (with only DPDD columns)
```

Comment:
Only DPDD columns as _non-native_
All columns necessary to create DPDD columns as _native_